src/phantom.js

```diff
@@ -45,7 +36,36 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        let loggerLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
+        if(typeof config.logger === 'object') {
```

Comment:
Yea, I didn't think about that. You are right. Hmm. I'd say let's just do it that way since it's all in one application.